java-api/src/main/java/com/sap/cloud/security/token/Token.java

```diff
@@ -26,6 +27,8 @@
  */
 public interface Token extends Serializable {
 
+	Logger LOGGER = LoggerFactory.getLogger(Token.class);
```

Comment:
Don't use LOGGER in interfaces :-)

Reply:
but then I cant fulfil the prerequisite in #668 logging error